Source/Particles/Collision/BinaryCollision/BinaryCollisionUtils.cpp

```diff
@@ -108,11 +108,27 @@ namespace BinaryCollisionUtils{
             amrex::ParmParse pp_collision_name(collision_name);
             std::string type;
             pp_collision_name.get("type", type);
+            amrex::Vector<std::string> product_species_name;
+            pp_collision_name.getarr("product_species", product_species_name);
+
             if (type == "nuclearfusion") {
                 NuclearFusionType fusion_type = get_nuclear_fusion_type(collision_name, mypc);
                 return nuclear_fusion_type_to_collision_type(fusion_type);
             }
-            amrex::Abort(type + " is not a valid type of collision that creates new particles");
+            if (type == "photonphoton") {
+                WARPX_ALWAYS_ASSERT_WITH_MESSAGE(
+                    product_species_name.size() == 2u,
+                    "ERROR: Photon-photon collisions must contain exactly two product species");
```

Comment:
```suggestion
                    "Photon-photon collisions must contain exactly two product species");
```
You don't need to write `ERROR: ` here. If the assert fails, the message is automatically prepended with `### ERROR   :`